src/app/views/NoteCreate.svelte

```diff
@@ -30,10 +48,18 @@
   let showPreview = false
   let showOptions = false
   let signaturePending = false
-
   let editor: Editor
   let element: HTMLElement
-  let options = {warning: "", anonymous: false}
+  let options = {warning: "", anonymous: false, publish_at: null}
+
+  const DVM_HANDLER_FILTER = {
+    kinds: [HANDLER_INFORMATION],
+    "#k": [DVM_REQUEST_PUBLISH_SCHEDULE.toString()],
+  }
+
+  const handlers = deriveEvents(repository, {
+    filters: [DVM_HANDLER_FILTER],
+  })
```

Comment:
Instead of loading handlers, let's just hardcode shipyard's pubkey (since we don't allow users to select which dvm to use anyway)